plugins/woocommerce/tests/e2e-pw/tests/shopper/product-variable.spec.js

```diff
@@ -203,9 +203,18 @@ test.describe( 'Variable Product Page', () => {
 		await expect(
 			page.locator( 'td.product-name >> nth=0' )
 		).toContainText( variableProductName );
-		await expect( page.locator( 'tr.order-total > td' ) ).toContainText(
-			( +productPrice * 10 ).toString()
+
```

Comment:
Going to assert that the price displayed is within a range.  It should be between the product price and the product price + 25% (the maximum tax rate that we test).  Depending on when this test runs, it may run when taxes are enabled.  Looking in this range will account for that.

Reply:
Going to accept this, but long-term, it'd be good to know for certain whether taxes were enabled, otherwise this assertion doesn't seem very useful to me.

Reply:
It should resolve around the time we can shard tests. The trouble is that the tests for taxes could be running at the same time as these tests and on the same store... but once we can get some isolation, then we should be good.